### PR TITLE
GSAGH-242 fix buckling length factors

### PR DESCRIPTION
**Pull request by kpne**

### CODE REVIEW

File: GsaGH/Parameters/1_Properties/GsaBucklingLengthFactorsGoo.cs
Comment (DominikaLos) [inline GsaGH/Parameters/1_Properties/GsaBucklingLengthFactorsGoo.cs:27]: goo without castFrom? it was intentional?

**Comment by tlmnrnhrdt** [comment]
Can we please apply what Tim said yesterday and not request review before a pr is ready? ;)

Which means we should get probably get rid off this "requested review from code owner"?!

File: GsaGH/Parameters/1_Properties/GsaBucklingLengthFactorsGoo.cs
Comment (kpne) [reply GsaGH/Parameters/1_Properties/GsaBucklingLengthFactorsGoo.cs:27]: Yes I didn’t want anyone to cast from a single double into three doubles, as the factors are unrelated and would never realistically all be the same. So removed the cast

**Comment by kpne** [comment]
> Can we please apply what Tim said yesterday and not request review before a pr is ready? ;)
> 
> Which means we should get probably get rid off this "requested review from code owner"?!

It’s ready though?

**Review by tlmnrnhrdt — APPROVED** [review]

**Comment by tlmnrnhrdt** [comment]
> 

Ok, if i had read your comment it might would have been lol

This way I spent 2 minutes looking at my own changes ;)

**Pull request merged**
